azext_iot/_help.py

```diff
@@ -576,6 +576,49 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Download, upload or migrate the state of an IoT Hub.
+"""
+
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Saves the configuration and device information of an IoT Hub to a file.
+    examples:
+    - name: Export the devices in the specified hub to the specified file.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename}
+"""
+
+helps[
+    "iot hub state import"
+] = """
+    type: command
+    short-summary: Uploads the configuration and device information from a file to an IoT Hub.
+    examples:
+    - name: Upload the devices from the specified file to the specified hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename}
+    - name: Upload the devices from the default file to the specified hub, overwriting the configurations and devices already in the hub.
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename} -r
+"""
+
+helps[
+    "iot hub state migrate"
+] = """
+    type: command
+    short-summary: Copies the configuration and device information from one hub to another hub without saving to a file.
+    examples:
+    - name: Migrate the state of the original hub to the new hub, overwriting the configurations and devices already in the hub.
```

Comment:
add an example without the -r flag

Reply:
also change this example (the one with -r) to show all the params - so you can show that you can do this with hubs in different rgs